### PR TITLE
Edit the README for tone and clarity

### PR DESCRIPTION
**Pull request by willingc**
Addresses #7.

Edits content to improve clarity and usability by sprint leaders. Improves user friendliness of document's tone.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

Unfortunately we couldn't find an account corresponding
to your GitHub username at [bugs.python.org](http://bugs.python.org/) (b.p.o).
If you don't already have an account at b.p.o, please
[create one](http://bugs.python.org/user?@template=register) and make sure to
add your GitHub username. If you do already have an account at b.p.o then
please go there and under "Your Details" add your GitHub username.

And in case you haven't already, please make sure to sign the
[PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
(CLA); we can't legally look at your contribution until you have signed the
CLA.

Once you have done everything that's needed, please reply here and someone will
verify everything is in order.

**Comment by willingc** [comment]
Well, the tool caught that I did not have a GitHub address in my bugs.python.org account even though I do have a signed CLA. I've updated the bugs.python.org account to include my GitHub user name.

I will push a small change to my PR branch and see if this resolves the CLA not signed tag.

**Comment by willingc** [comment]
Ooh. Very cool. The tool did change to CLA signed tag on a push to the branch. Nicely implemented.

Perhaps consider changing the default error message when there is no GitHub address in the bugs.python.org account so that a contributor can resolve themselves and not require interaction from a developer.

File: README.rst [hunk 1]
```diff
@@ -54,20 +66,18 @@ update repositories or generating patches.
...
 sure to install the command-line tools (e.g. in Mavericks,
 ``xcode-select --install``).
 
-If new contributors think they may be doing C development, suggest LLVM + clang
-for better error reporting than gcc.
+If new contributors think they may be doing C development, suggest the use of
+LLVM + clang as this provides better error reporting than gcc.
 
-For Windows users, tell them to download and install Visual C++ Express
+For Windows users, ask them to download and install Visual C++ Express
```
Comment (ncoghlan) [inline README.rst:80]: Hmm, this is outdated (it's Visual Studio Community edition now)

I'll fix that directly after merging.

File: README.rst [hunk 2]
```diff
@@ -78,34 +88,31 @@ Clone the `CPython repository`_ and build it (you will be cleaning up your build
...
-  # Assuming at the top of the devinabox.
+  # Assuming at the root of the devinabox directory
   python build_cpython.py
   ./cpython/python -m venv venv
   ./venv/bin/pip install sphinx
   cd cpython/Doc
   make html PYTHON=../../venv/bin/python
 
-.. _CPython repository: http://hg.python.org/cpython
+.. _CPython repository: https://github.com/python/devinabox
```
Comment (ncoghlan) [inline README.rst:104]: This link target is for the CPython repo, which hasn't moved yet, rather than the devinabox repo itself

**Comment by ncoghlan** [comment]
Couple of fix-ups suggested, but I'll do those myself post-merge.

**Comment by willingc** [comment]
Thanks @ncoghlan. Also thank you to who wrote the CLA tool. It worked out very well :smile:

**Pull request merged**